WalletWasabi/WabiSabi/Client/WabiSabiHttpApiClient.cs

```diff
@@ -67,15 +71,52 @@ public async Task<RoundState[]> GetStatusAsync(CancellationToken cancellationTok
 		public Task ReadyToSignAsync(ReadyToSignRequestRequest request, CancellationToken cancellationToken) =>
 			SendAndReceiveAsync<ReadyToSignRequestRequest>(RemoteAction.ReadyToSign, request, cancellationToken);
 
+		private async Task<HttpResponseMessage> SendWithRetriesAsync(RemoteAction action, string jsonString, CancellationToken cancellationToken)
+		{
+			var errors = new List<Exception>();
+
+			var start = DateTime.Now;
+
+			for (var attempt = 0; attempt < MaxRetries; attempt++)
+			{
+				try
+				{
+					using StringContent content = new(jsonString, Encoding.UTF8, "application/json");
+
+					// Any transport layer errors will throw an exception here.
+					var response = await _client.SendAsync(HttpMethod.Post, GetUriEndPoint(action), content, cancellationToken).ConfigureAwait(false);
```

Comment:
```suggestion
					using var response = await _client.SendAsync(HttpMethod.Post, GetUriEndPoint(action), content, cancellationToken).ConfigureAwait(false);
```

Reply:
if there's a `using` doesn't it mean it will be disposed before it's returned?

Reply:
Yes, sorry.